docs/guides/postinstall.md

```diff
@@ -185,58 +185,59 @@ S3 suspend has been broken since macOS Sonoma, it has never been fixed, but this
 
 5. If you are facing issues with Wi-Fi on resume, uncomment the lines having `brcmfmac` and `brcmfmac_wcc` in the above file.
 
-> **Note:**
+> !!! note
     Make sure you have `CONFIG_MODULE_FORCE_UNLOAD=y` in the kernel config.
     To check, run: `zcat /proc/config.gz | grep "CONFIG_MODULE_FORCE_UNLOAD"` on Arch based distros.
 
-
-
-
 ### Gentoo/OpenRC
 
 For Gentoo Linux using OpenRC and elogind, create the following script:
 
-While it's technically sufficient to unload only the apple-bce module for suspend to work, this will result in a non-functional Touch Bar after resume. This workaround ensures the Touch Bar is properly reinitialized by also handling the Touch Bar specific modules and the tiny-dfr service.
+For T2 MacBooks, while unloading only the apple-bce module is sufficient for basic suspend functionality, additional module handling may be required depending on your model:
+
+- All T2 models require the apple-bce module handling
+- For models with Touch Bar, a specific module sequence (apple_bce -> hid_appletb_bl -> hid_appletb_kbd -> appletbdrm) is required to properly reinitialize the Touch Bar device after resume
```

Comment:
I'm still not convinced with the hid_appletb_kbd bit. Please share the outputs requested as mentioned in the first review

Reply:
Disregard my previous statement, I did a few tests on my end, and hid_appletb_kbd seems to be needed still (not sure why though). So, I'm resolving this.